server/views/pages/activities/activitiesMigration.njk

```diff
@@ -156,25 +184,35 @@
 
                 <div class="govuk-grid-row govuk-!-margin-bottom-2">
                   <div class="govuk-grid-column-full">
-                    {% set endNomisLink = "/activities-migration/end-activities?migrationId=" + u.migrationId %}
-                    {{ govukButton({
-                      text: "End NOMIS activities on " + u.filterEndDate,
-                      href: endNomisLink,
-                      isStartButton: false,
-                      attributes: { 'data-qa': 'end-activities-button' }
-                    }) }}
-                    {% if endMigratedActivitiesResult.migrationId == u.migrationId %}
-                      {% if endMigratedActivitiesResult.result == 'OK' %}
-                        {% set resultClass = 'result-success' %}
-                      {% else %}
-                        {% set resultClass = 'result-error' %}
+                    {% if u.dateAction == 'END' %}
+                      {% set endNomisLink = "/activities-migration/end-activities?migrationId=" + u.migrationId %}
+                      {{ govukButton({
+                        text: "End NOMIS activities on " + u.filterEndDate,
+                        href: endNomisLink,
+                        isStartButton: false,
+                        attributes: { 'data-qa': 'end-activities-button' }
+                      }) }}
+                      {% if endMigratedActivitiesResult.migrationId == u.migrationId %}
+                        {% if endMigratedActivitiesResult.result == 'OK' %}
+                          {% set resultClass = 'result-success' %}
+                        {% else %}
+                          {% set resultClass = 'result-error' %}
+                        {% endif %}
+                        <span class="{{ resultClass }}" data-qa="end-activities-result">
+                                  {{ endMigratedActivitiesResult.result }}
+                                </span>
                       {% endif %}
-                      <span class="{{ resultClass }}" data-qa="end-activities-result">
-                                {{ endMigratedActivitiesResult.result }}
-                              </span>
+                    {% elseif u.dateAction == 'MOVE' %}
+                      {% set moveStartDateLink = "/activities-migration/move-start-date/start?migrationId=" + u.migrationId %}
+                      {{ govukButton({
+                        text: "Move start date",
+                        href: moveStartDateLink,
+                        isStartButton: false,
+                        attributes: { 'data-qa': 'start-move-start-date-button' }
+                      }) }}
```

Comment:
This is the main change - we've added a button that allows the dev / BA doing the migration to move the activity start date forwards. This allows the activity team to rollout faster because prisons are less reluctant to sign up for DPS activities if they can push the go live back rather than being tied to a specific date.